chapter_optimization/momentum_vn.md

```diff
@@ -171,7 +175,12 @@ The example below illustrates what happens even after a slight increase in learn
 Convergence in the $x_1$ direction improves but the overall solution quality is much worse.
 -->
 
-*dịch đoạn phía trên*
+Ta thấy, gradient theo hướng $x_2$ có giá trị *lớn hơn nhiều* trong khi gradient theo hướng ngang $x_1$ thay đổi giá trị nhanh hơn.
+Vì thế, chúng ta bị mắc kẹt giữa hai lựa chọn không mong muốn: Nếu chúng ta chọn một tốc độ học nhỏ, chúng ta sẽ đảm bảo rằng các nghiệm sẽ không phân kỳ theo hướng $x_2$
+nhưng chúng ta sẽ rơi vào các điểm yên ngựa với tốc độ hội tụ chậm theo hướng $x_1$.
```

Comment:
```suggestion
nhưng tốc độ hội tụ sẽ chậm theo hướng $x_1$.
```
chỗ này `saddled` nghĩa là `struggled`, ko phải `saddle points`.

Reply:
Anh tra thì ra `To get into a saddle`. Anw, dịch ngắn gọn vậy cũng k làm méo nghĩa gốc :D